<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSMclient</class>
 <widget class="QMainWindow" name="LSMclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSMclient</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.jpg</normaloff>icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="window">
   <widget class="QPushButton" name="scan_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SCAN</string>
    </property>
   </widget>
   <widget class="QSlider" name="x_offset_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="x_offset_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X offset:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_offset_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y offset:</string>
    </property>
   </widget>
   <widget class="QSlider" name="y_offset_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="x_res_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X resolution:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_resolution_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_resolution_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="y_res_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y resolution:</string>
    </property>
   </widget>
   <widget class="QSlider" name="zoom_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="zoom_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="laser_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser intensity:</string>
    </property>
   </widget>
   <widget class="QSlider" name="laser_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="gain_combo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64x</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="gain_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amplification:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coarse_button">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Coarse focus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fine_button">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Fine focus</string>
    </property>
   </widget>
   <widget class="QLabel" name="avg_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC average:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="avg_combo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="image_view">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
   </widget>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
